metron-platform/metron-common/src/main/java/org/apache/metron/common/message/JSONFromPosition.java

```diff
@@ -40,10 +40,12 @@ public JSONFromPosition(int position) {
 
   @Override
   public JSONObject get(Tuple tuple) {
+    String s = null;
     try {
-      return (JSONObject) parser.get().parse(new String(tuple.getBinary(position), "UTF8"));
+      s =  new String(tuple.getBinary(position), "UTF8");
```

Comment:
`UTF-8` and `UTF8` are aliases in the `StandardCharsets` from looking through the java API code.